project-base/storefront/pages/search.tsx

```diff
@@ -48,12 +54,20 @@ export const getServerSideProps = getServerSidePropsWrapper(({ redisClient, doma
         return redirect;
     }
 
-    return initServerSideProps({
-        context,
-        redisClient,
-        domainConfig,
-        t,
-    });
+    return initServerSideProps<TypeSearchQueryVariables | TypeSearchProductsQueryVariables | TypeAdvertsQueryVariables>(
+        {
+            context,
+            redisClient,
+            domainConfig,
+            t,
+            prefetchedQueries: [
+                {
+                    query: AdvertsQueryDocument,
+                    variables: { positionName: 'productListSecondRow', categoryUuid: null },
```

Comment:
is the advert even rendered on the search page?

I have tried to create the advert for this position and in the wishlist is not rendered on the 15.0 nor on this branch

Reply:
Indeed it is not rendered. I went only by references and didn't realize that we display the adverts only if we receive a category in the props of `ProductsList` component.